src/prefect/engine/cloud/flow_runner.py

```diff
@@ -182,9 +182,9 @@ def interrupt_if_cancelling() -> None:
                             "Error getting flow run info", exc_info=True
                         )
                         continue
-                    else:
-                        self.logger.debug(
-                            "Successfully queried server for flow run state: %r",
+                    if not flow_run_info.state.is_running():
+                        self.logger.warning(
+                            "Flow run is no longer in a running state; the current state is: %r",
```

Comment:
Is this useful to see? Task runs should eventually bubble back to shutdown the flow runner, so checking for running here isn't necessary (but may still be useful?).

Reply:
Good question - I think it is; there are two situations I've seen where this log would be nice:
- users whose task runs error out when they attempt to enter a `Running` state because the flow run state is already finished ("cancellation lite"); for some reason many users have a hard time debugging this, and I think seeing this log might help tell the story
- earlier @zhen0 identified a weird race condition where the flow run was running but the backend thought it was in a `Submitted` state, and this log was one way she identified that